app/commands.py

```diff
@@ -115,3 +115,33 @@ def update_notification_international_flag(self):
             print('commit 250000 updates at {}'.format(datetime.utcnow()))
             db.session.commit()
             result_history = db.session.execute(subq_history).fetchall()
+
+    def fix_notification_statuses_not_in_sync(self):
+        """
+        This will be used to correct an issue where Notification._status_enum and NotificationHistory._status_fkey
+        became out of sync. See 979e90a.
+
+        Notification._status_enum is the source of truth so NotificationHistory._status_fkey will be updated with
+        these values.
+        """
+        MAX = 10000
```

Comment:
What happens when `len(result)` is less and `MAX`? 
Wouldn't the message be misleading?